dev/TabView/TabView.cpp

```diff
@@ -350,14 +350,49 @@ void TabView::OnScrollViewerLoaded(const winrt::IInspectable&, const winrt::Rout
 
         auto increaseButton = SharedHelpers::FindInVisualTreeByName(scrollViewer, L"ScrollIncreaseButton").as<winrt::RepeatButton>();
         m_scrollIncreaseClickRevoker = increaseButton.Click(winrt::auto_revoke, { this, &TabView::OnScrollIncreaseClick });
+
+        scrollViewer.ViewChanged({ this, &TabView::OnScrollViewerViewChanged });
```

Comment:
store the return value in a revoker similar to the line above.

Reply:
Done, please check latest iteration.